app-web/plugins/gatsby-source-github-all/utils/constants.js

```diff
@@ -27,6 +27,11 @@ const SOURCE_TYPES = {
   GITHUB: 'github',
 };
 
+const COLLECTION_TYPES = {
+  [SOURCE_TYPES.GITHUB]: 'source',
```

Comment:
Nice trick. I didn't know you could do that. 🥇

Reply:
awh yeaahhh